test/integration/integration_test.go

```diff
@@ -48,6 +53,10 @@ var (
 
 	testDir string
 
+	//proxy
+	proxyListener net.Listener
+	proxyLog      *bytes.Buffer
```

Comment:
@hferentschik 

> I would extract all proxy related functions into a separate file proxy.go or something

I am notorious one-filer :( but I agree, will move it later into separate file after CI problem is resolved as @gbraad suggests.